Terminal.py

```diff
@@ -63,8 +63,8 @@ def linux_terminal():
     if wm:
         # elementary OS: `/usr/lib/gnome-session/gnome-session-binary --session=pantheon`
         # Gnome: `gnome-session` or `gnome-session-binary`
-        # Linux Mint Cinnamon: `cinnamon-session --session cinnamon`
-        if wm[0].startswith('gnome-session') or wm[0].startswith('cinnamon-session'):
+        # Linux Mint Cinnamon: `cinnamon-sessio cinnamon-session --session cinnamon`
+        if wm[0].startswith('gnome-session') or wm[0].startswith('cinnamon-sessio'):
```

Comment:
The proper fix to me sounds as if the grep call should be fixed. Would the detection still work if the `n` was added in line 61 instead?

Reply:
The `grep` is working fine since we're getting the line back for comparison in Python (i.e. `grep` doesn't truncate lines, it filters them)

The `n` truncation is coming from `ps` itself, which might be an OS-level truncation.

It's probably sanest to use the implementation I've proposed as anything else (e.g. changing `ps` output), would likely result in cascading changes needed for all the other environments